src/Tribe/Events/Views/V2/Hooks.php

```diff
@@ -89,7 +122,8 @@ protected function add_actions() {
 	 * @since 4.10.9
 	 */
 	protected function add_filters() {
-		add_filter( 'tribe_template_path_list', [ $this, 'filter_template_path_list' ] );
+		add_filter( 'tribe_template_path_list', [ $this, 'filter_template_path_list' ], 15, 2 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: ea383631ac758cc8a42a0429e1a810c8</sub>